src/main/java/ulcrs/controllers/SessionController.java

```diff
@@ -14,36 +25,82 @@
 
 public class SessionController extends BaseController {
 
+    private static final String WORKSPACE_PATH = "workspace/";
+
     @Override
     public RouteGroup routes() {
         return () -> {
             before("/*", (request, response) -> log.info("endpoint: " + request.pathInfo()));
-            get("/", this::getSessionList, gson::toJson);
+            get("/", this::getSessionList, sessions -> {
+                Gson gson = new GsonBuilder().excludeFieldsWithoutExposeAnnotation().setPrettyPrinting().create();
```

Comment:
If using the GsonFactory, I believe this and the following lines can be handled by `gson::toJson`

Reply:
Changed to exposeOnlyGson